<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displayTeachers</class>
 <widget class="QDialog" name="displayTeachers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>871</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>First name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>phone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>dateOfBirth</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>specialization</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Delete_teacher">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Edit_teacher">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Assign_courses">
      <property name="text">
       <string>Display/Edit assinged courses</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
